### ISSUE TITLE
ChoiceEndChoice constant expression result

### ISSUE THREAD

**Issue by czchen** [chewing/libchewing#247]
The following expression is always true. Need to check if it is a bug or not.

```c
    if (pgdata->choiceInfo.isSymbol != WORD_CHOICE || pgdata->choiceInfo.isSymbol != SYMBOL_CHOICE_INSERT) {
    ...
```

**Comment by Billy4195** [comment]
After tracing the history of choice.c, in [f936abe9](https://github.com/chewing/libchewing/commit/f936abe9908a2f1487f174980029180e017cc219) the check is want to check if the choice is symbol, if it is word select then cursor will return to old cursor position.
But, later another [commit](https://github.com/chewing/libchewing/commit/930d1f63a29820b7e0d65c953518a99e27842163), jserv add the symbol check, after this commit the expression is true in any case.
then The [commit](https://github.com/chewing/libchewing/commit/9c6a064af7ea3d4e45bb33d453a30eae0f2fc7c8) change number to symbolic name, and the replacement has some mistake(I have add comment in this commit).

From the above observation, this field of code is like to handle cursor back to old place after choice finished, original commit only handle word select case, but later commit add symbol as well.
In conclusion, the cursor should point back to original position after finishing choice, no matter it is word or symbol choice. 
I suggest to remove the if statement, let cursor return to old position in any case.

**Comment by Billy4195** [comment]
Refer to #258, this issue can be closed.
